torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -447,6 +449,24 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
     }
   }
 
+  if (py_module_.attr("_constants_set").contains(field.c_str())) {
+    // Values of the attributes listed in the _constants_set will be put
+    // directly into IR. In order to allow us to access these values by their
+    // name after IR is generated, we register them as attributes.
+    if (py::isinstance<py::bool_>(attr)) {
```

Comment:
`pybind_utils.h` has the functions `tryToInferType` and `toIValue`, which can be combined to make this more general.